### PR TITLE
Update tutorials and add distance 

### PR DESCRIPTION
**Pull request by cameronmartino**
**Updates:**

* Added a distance output of type skbio.stats.distance.DistanceMatrix
* Updated the documentation to add PERMANOVA on distance output
* Added links to qiime2 plugin site in the readme
* Changed the plugin run name from rpca-biplot to just rpca 
* Updated testing to reflect function name changes
*  Updated both plugin docs and .md in repo to reflect function name changes
* Push version number up by one

### CODE REVIEW

File: ipynb/sleep_apnea/SleepApnea-qiime2-tutorial.ipynb [hunk 1]
```diff
@@ -1,54 +1,190 @@
...
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "First we will convert the biom file to a Qiime2 style qza table"
+    "The first are sample filtering cutoffs these are  p-min-feature-count and p-min-sample-count. Both of these parameters accept integer values and remove feature or samples, respectively, with sums below this cutoff. The feature cut-off is useful in the case that features with very low total counts among all samples represent contamination or chimeric sequences. The sample cut off is useful for the case that some sample received very few reads relative to other samples.\n",
+    "\n",
+    "**Note:** it _not_ recommended to bin your features by taxonomic assignment (i.e. by genus level). <br>\n",
```
Comment (ElDeveloper) [inline ipynb/sleep_apnea/SleepApnea-qiime2-tutorial.ipynb]: it not -> it is not (same for next line).

File: deicode/q2/_method.py [hunk 2]
```diff
@@ -6,10 +6,10 @@
 from deicode.optspace import OptSpace
 from deicode.preprocessing import rclr
 
-def rpca_biplot(table: biom.Table, rank: int=3, 
+def rpca(table: biom.Table, rank: int=3,
```
Comment (ElDeveloper) [inline deicode/q2/_method.py]: Looks like flake8 is not running as part of the CI build, would be good to include that in a dedicated PR.

File: README.md [hunk 3]
```diff
@@ -1,81 +1,35 @@
...
-It should look as follows
-![biplot](https://github.com/biocore/DEICODE/blob/master/ipynb/sleep_apnea/qiime_view.png)
+## Other Resources
 
-### Python Tutorial
+* [What is Robust Aitchison RPCA](https://github.com/biocore/DEICODE/blob/master/ipynb/introduction.ipynb)
 
-* [Sleep Apnea Log Ratio Tutorial](https://github.com/biocore/DEICODE/blob/master/ipynb/sleep_apnea/SleepApnea-python-tutorial.ipynb)
+- The code for OptSpace was translated to python from a [MATLAB package](http://swoh.web.engr.illinois.edu/software/optspace/code.html) maintained by Sewoong Oh (UIUC).
+- Transforms and PCoA : [Scikit-bio](https://github.com/biocore/scikit-bio)
```
Comment (ElDeveloper) [inline README.md]: I would link to http://scikit-bio.org instead.

**Review by ElDeveloper — APPROVED** [review]
Looks good.

This is minor but I would suggest using DEICODE consistently throughout the tutorial, as well as QIIME instead of Qiime.

In some places you refer to q2-deicode, but that really is just installing DEICODE in the context of QIIME2. Maybe you can clarify that?

File: ipynb/sleep_apnea/SleepApnea-qiime2-tutorial.ipynb (hunk 1 above)
Comment (cameronmartino) [reply ipynb/sleep_apnea/SleepApnea-qiime2-tutorial.ipynb]: fixed, thank you!

File: deicode/q2/_method.py (hunk 2 above)
Comment (cameronmartino) [reply deicode/q2/_method.py]: added this to travis! very helpful.

File: README.md (hunk 3 above)
Comment (cameronmartino) [reply README.md]: @ElDeveloper changed nice catch. thanks!

**Comment by cameronmartino** [comment]
@ElDeveloper  I can live with the -.1% in coverage .... should not affect anything. Probably from flake8 edits.

**Comment by ElDeveloper** [comment]
Sounds good, then this should be good to go!

**Pull request merged**